site/_posts/2018-07-13-parameter-server-in-fifteen-lines.markdown

```diff
@@ -12,11 +12,12 @@ that process data and compute updates to the parameters).
 
 Parameter servers (like databases) are normally built and shipped as standalone
 systems. This post describes how to use [Ray][1] to implement a parameter server
-in a few lines of code. This is powerful for two reasons:
+in a few lines of code. This approach is powerful for two reasons:
 
-1. It is orders of magnitude simpler to deploy applications that use parameter
-servers.
-2. The behavior of the parameter server is much more configurable and flexible.
+1. This approach makes it orders of magnitude simpler to deploy applications
```

Comment:
I still think these points are a bit unclear -- orders of magnitude simpler than what? from scratch python?